renku-model/src/test/scala/io/renku/graph/model/entities/ActivityLensSpec.scala

```diff
@@ -0,0 +1,39 @@
+package io.renku.graph.model.entities
+
+import io.renku.graph.model.RenkuUrl
+import io.renku.graph.model.projects.DateCreated
+import io.renku.graph.model.testentities.generators.{ActivityGenerators, EntitiesGenerators}
+import org.scalatest.matchers.should
+import org.scalatest.wordspec.AnyWordSpec
+
+import java.time.Instant
+
+class ActivityLensSpec extends AnyWordSpec with should.Matchers {
```

Comment:
I think, if you'd import `io.renku.graph.model.testentities._` you could remove the implicit `renkuUrl` val as well as just use the `activityEntities`, `planEntities` and other without necessity of reaching to relevant generator classes.

Reply:
Thanks! 👍🏼